client-mvp-04/src/store/configureStore.js

```diff
@@ -1,10 +1,9 @@
 import { createStore } from 'redux';
-import allReducers from '../reducers/index';
+import allReducers from '../reducers';
 
 export default () => {
-    // Store creation
     const store = createStore(
-        allReducers, /* preloadedState, */
+        allReducers,
         window.__REDUX_DEVTOOLS_EXTENSION__ && window.__REDUX_DEVTOOLS_EXTENSION__()
     );
     return store;
```

Comment:
I still think the `const store` is an unnecessary variable here.
Please, share your opinion in the comment below. Thanks.

```
export default () => {
    return createStore(
        allReducers,
        window.__REDUX_DEVTOOLS_EXTENSION__ && window.__REDUX_DEVTOOLS_EXTENSION__()
    );
}

```